packages/data-service/src/connect-mongo-client.spec.ts

```diff
@@ -1,13 +1,16 @@
 import assert from 'assert';
 import asyncHooks from 'async_hooks';
 import { expect } from 'chai';
-import type { CommandStartedEvent } from 'mongodb';
+import type { MongoClientOptions } from 'mongodb';
+import { type CommandStartedEvent } from 'mongodb';
```

Comment:
Probably a nit: What's with this style? Can't CommandStartedEvent be imported along with MongoClientOptions in the previous line?

Reply:
Huh, I’m surprised myself :slightly_smiling_face: This was just what VSCode automatically did for me when I added the usage of `MongoClientOptions`, not something I intentionally added. Yes, they should be merged imo.